language/wrappers/php.xml

```
@@ -368,6 +368,31 @@ file_put_contents("php://filter/write=string.rot13/resource=example.txt","Hello
 <?php
 file_put_contents('php://memory', 'PHP');
 echo file_get_contents('php://memory'); // prints nothing
+]]>
+   </programlisting>
+  </example>
+  <example>
+   <title>php://input to read JSON data from the request body</title>
+   <para>
+    This example demonstrates how to read JSON data from POST, PUT and PATCH
+    requests using <filename>php://input</filename>.
+   </para>
+   <programlisting role="php">
+<![CDATA[
+<?php
+header('Content-Type: text/plain');
+
+// Read raw input data from php://input
+$jsonData = file_get_contents("php://input");
+
+// Decode JSON data into a PHP array
+$data = json_decode($jsonData, true);
```

Use [JSON_THROW_ON_ERROR](https://www.php.net/manual/en/json.constants.php#constant.json-throw-on-error) instead of checking the return value.